lib/transformer.js

```diff
@@ -177,11 +177,9 @@ class Transformer {
     for (const obj of interfaces.values()) {
       let source = obj.toString();
 
-      const implDir = path.relative(outputDir, obj.opts.implDir).replace(/\\/g, "/"); // fix windows file paths
-      let implFile = implDir + "/" + obj.name + this.ctx.implSuffix;
-      if (implFile[0] !== ".") {
-        implFile = "./" + implFile;
-      }
+      const implDir = obj.opts.implDir.replace(/\\/g, "/"); // fix windows file paths
+      let implFile = path.relative(
+        outputDir, path.resolve(implDir, obj.name + this.ctx.implSuffix));
```

Comment:
We wrap at 120 cols, so these lines could be coalesced.

Reply:
Done.